code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -615,111 +615,6 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks)
 		consumer.urine += 0.1
 		eat_twitch(consumer)
 
-	//bleck, i dont like this at all. (Copied from chemistry-tools reagent_containers/glass/ definition w minor adjustments)
-	// still copy paste btw
-	afterattack(obj/target, mob/user , flag)
-		if (is_sealed)
-			boutput(user, "<span class='alert'>[src] is sealed.</span>")
```

Comment:
why is this deleted

Reply:
i moved the code to obj/item/reagent_containers and thus make it inherit the code, no need for 2 blocks of the same code